draft-packages/pagination/KaizenDraft/Pagination.tsx

```diff
@@ -0,0 +1,168 @@
+import React from "react"
+import arrowBackward from "@kaizen/component-library/icons/arrow-backward.icon.svg"
+import arrowForward from "@kaizen/component-library/icons/arrow-forward.icon.svg"
+import { Icon } from "@kaizen/component-library"
+import cx from "classnames"
+import PageIndicator from "./PageIndicator"
+import TruncateIndicator from "./TruncateIndicator"
+import styles from "./styles.scss"
+
+export const createRange = (start: number, end: number): number[] => {
+  const range: number[] = []
+  for (let i = start; i <= end; i++) {
+    range.push(i)
+  }
+  return range
+}
+
+type Props = {
+  currentPage: number
+  pageCount: number
+  ariaLabelNextPage?: string
+  ariaLabelPreviousPage?: string
+  ariaLabelPage?: string
```

Comment:
Similar to Dan's comment on `PageIndicator`, these aria label props should be required I'd imagine.